packages/client/src/pages/Room.tsx

```diff
@@ -76,7 +81,10 @@ export const Room = () => {
               <FiArrowLeftCircle title="Back to Lobby" size={24} color={theme.colors.text} />
             </Link>
           </Box>
-          <Game gameState={gameState} gameScore={gameScore} />
+          <Flex sx={{ justifyContent: 'space-between', height: '100%' }}>
+            <Game sx={{ flex: '1 1 auto' }} gameState={gameState} gameScore={gameScore} />
+            <Log event={actionDescription} />
```

Comment:
I think you can do `sx={{ flex: '1 1 25%' }}` or well play with the flex property and stick to this instead of having width on the component at least it's clear looking at it.
Or I would use grid as parent and specify columns there
In both cases we need to think what to do for mobile

Reply:
When can I do that, on `Log`? If so it doesn't seem to work...

Reply:
because you need to spread the props in the component then :-p look at Game